src/design.py

```diff
@@ -140,12 +142,12 @@ def __init__(self, system, keys=keys, ranges=ranges,labels=labels, array = desig
         #   - wrap normal text with \mathrm{}
         #   - escape spaces
         #   - surround with $$
-        self.labels = [
-            re.sub(r'({[A-Za-z]+})', r'\\mathrm\\1', i)
-            .replace(' ', r'\ ')
-            .join('$$')
-            for i in labels
-        ]
+        #self.labels = [
+        #    re.sub(r'({[A-Za-z]+})', r'\\mathrm\\1', i)
+        #    .replace(' ', r'\ ')
+        #    .join('$$')
+        #    for i in labels
+        #]
 
```

Comment:
It seems like this got lost along the way? Was this causing problems? Maybe it's good to have an option not to apply any formatting, but I wouldn't remove it by default

Reply:
Yeah I commented this out while I was testing stuff because it threw a fit and forgot to uncomment it later.

Reply:
I'm not very experienced with githubs's interface, how do I revert the change for the pull request?

Reply:
I would just create a new commit on this branch that removes the commenting out. I can then squash all of the commits together when we merge it